### ISSUE TITLE
Tooltip hide/show is buggy and strobes rapidly

### ISSUE THREAD

**Issue by stantonk** [28mm/blast-radius#11]
I started up blast-radius using the instructions in the README with a very basic Terraform setup from the [introductory tutorial.](https://www.terraform.io/intro/getting-started/build.html).

The tooltip strobes on/off rapidly. So fast that it is somewhat hard to see in this gif:

![blast-radius-tooltip-bug](https://user-images.githubusercontent.com/519545/35082281-8a0f3554-fbde-11e7-82d5-56ca4a44349b.gif)

**Comment by stantonk** [comment]
Better gif:

![blast-radius-tooltip-bug](https://user-images.githubusercontent.com/519545/35082329-bdadb908-fbde-11e7-810b-b5418f5e0596.gif)

**Comment by 28mm** [comment]
Thanks for reporting—and the gifs are very helpful. this is likely down to some naive javascripting on my part.

Going to have a closer look tomorrow, but a temporary workaround is to click rather than hover. 

Thanks!
Patrick.

**Comment by stantonk** [comment]
Oh cool! Thanks for the tip on clicking, that is a great workaround in the meantime.

This is really exciting work. Our Terraform repo is _quite_ large and I have been looking for something to visualize. Unfortunately the way our repo is laid out the graph generation and SVG is intractable (takes 10+ minutes and pegs a CPU), we need to clean that up.

I'd love to see a tool someday producing a graph of servers from terraform configuration keyed on hostname to input to something like [vizceral](https://github.com/Netflix/vizceral) in concert with event data from [opentracing.io](http://opentracing.io/) or perhaps the newly open-sourced [OpenCensus](https://twitter.com/opencensusio) from Google.

**Comment by 28mm** [comment]
@stantonk -- bumping the tooltip offset up a few pixels seems to resolve the flicker issue. Without a small offset, a lot of conflicting mouse events fire and create the strobe effect... Anyway I've pushed the changes to the PyPi and Docker repositories.

I'm really interested in your repo. Most of my projects have been small-ish, and of course most public examples are as well. If you don't mind sharing, how many edges are we talking about?

`terraform graph | grep -c '\->'`

And just so I'm clear, are you saying that `terraform graph | dot -Tsvg` takes ten minutes, or is that fast and *blast radius* takes ten minutes?

I've opened issue #10 to add a few features to make it easier to get just a particular subgraph, and collapse module display-- same as *Terraform's* `--module-depth` option. I'm hoping those sorts of features will help in scenarios like you describe.

P.S. thanks for the links to vszceral, and opentracing... I agree that there could be some cool integrations there!
